src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Tcp/ClientCredentialTypeTests.cs

```diff
@@ -174,4 +173,47 @@ public static void TcpClientCredentialType_Certificate_EchoString()
             ScenarioTestHelpers.CloseCommunicationObjects((ICommunicationObject)serviceProxy, factory);
         }
     }
+
+    [Fact]
+    [OuterLoop]
+    public static void TcpClientCredentialType_Certificate_CustomValidator_EchoString()
+    {
```

Comment:
Adding the negative variation actually showed that the Validate method was not even being called, so the positive test was showing a false positive. I updated the test to confirm that the Validate method actually gets called no matter what.